plugins/plugins_test.go

```diff
@@ -264,8 +268,8 @@ func (m *mockForInitStartOrdering) Start(ctx context.Context) error {
 	return fmt.Errorf("expected manager to be initialized")
 }
 
-func (m *mockForInitStartOrdering) Stop(ctx context.Context)                            { return }
-func (m *mockForInitStartOrdering) Reconfigure(ctx context.Context, config interface{}) { return }
+func (m *mockForInitStartOrdering) Stop(ctx context.Context)                            {}
+func (m *mockForInitStartOrdering) Reconfigure(ctx context.Context, config interface{}) {}
```

Comment:
[nit] I'd take it on step further,
```suggestion
func (*mockForInitStartOrdering) Stop(context.Context)                     {}
func (*mockForInitStartOrdering) Reconfigure(context.Context, interface{}) {}
```

But it's OK as-is, too, of course.

Reply:
Nice!